tests/test_model.py

```diff
@@ -110,46 +110,40 @@ def test_CallableNumericalModel():
 
     xdata = np.linspace(0, 10)
     ydata = model(x=xdata, a=5.5, b=15.0).y + np.random.normal(0, 1)
+
+    # TODO #assert model(x=xdata, a=5.5, b=15.0) == pytest.approx(numerical_model(x=xdata, a=5.5, b=15.0))
     np.testing.assert_almost_equal(
         model(x=xdata, a=5.5, b=15.0),
-        numerical_model(x=xdata, a=5.5, b=15.0),
+        numerical_model(x=xdata, a=5.5, b=15.0)
     )
```

Comment:
I like the TODO in case this doesn't work, but could you still try `assert np.array(model(...)) == approx(np.array(numerical_model(...))`